Readme.md

```diff
@@ -77,6 +77,8 @@ properties for consistency.
 
 ## AST
 
+Interatively explore the AST with [http://iamdustan.com/reworkcss_ast_explorer/](http://iamdustan.com/reworkcss_ast_explorer/).
+
```

Comment:
Thanks for the tip, @lydell! Do you know off-hand if that standard markdown or github extensions?

Do you want me update that before landing this?